content/sklearn/concepts/kernel-ridge-regression/kernel-ridge-regression.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Kernel Ridge Regression'
+Description: 'Kernel ridge regression is a sophisticated linear regression model combined with L2 regularization and kernel trick to provide optimal solutions.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'Machine Learning'
+  - 'Python'
+  - 'Scikit-learn'
+CatalogContent:
+  - 'paths/intermediate-machine-learning-skill-path'
+---
+
+**Kernel ridge regression** is a regression model that combines ridge regression with the kernel trick.
+
+## Ridge Regression
+
+**Ridge regression** is a linear regression model with a least square loss function and L2 regularization. The loss function, equivalent to the sum of the coefficients squared, allows the linear regression model to find the best fit line on the training data as shown in the example below:
```

Comment:
``` suggestion
**Ridge regression** is a linear regression model with a least square loss function and L2 regularization. The loss function combines the; least square loss with the L2 regularization term and allows the linear regression model to find the best-fit line on the training data. It is shown in the example below:
```
Ridge regression is not just equivalent to the sum of the coefficients squared. It actually combines the least squares loss (which is the sum of the squared differences between the predicted and actual values) with the L2 regularization term.